pkg/targets/reconciler/awseventbridgetarget/eventbridge_reconciler.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2022 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package awseventbridgetarget
+
+import (
+	"context"
+
+	pkgreconciler "knative.dev/pkg/reconciler"
+
+	awsv1alpha1 "github.com/triggermesh/triggermesh/pkg/apis/targets/v1alpha1"
+	reconcilers "github.com/triggermesh/triggermesh/pkg/client/generated/injection/reconciler/targets/v1alpha1/awseventbridgetarget"
+	libreconciler "github.com/triggermesh/triggermesh/pkg/targets/reconciler"
+)
+
+// Reconciler reconciles the target adapter object
+type eventBridgeReconciler struct {
+	ksvcr libreconciler.KServiceReconciler
+	vg    libreconciler.ValueGetter
+
+	adapterCfg *adapterConfig
+}
+
+// Check that our Reconciler implements Interface
+var _ reconcilers.Interface = (*eventBridgeReconciler)(nil)
+
+// ReconcileKind implements Interface.ReconcileKind.
+func (r *eventBridgeReconciler) ReconcileKind(ctx context.Context, trg *awsv1alpha1.AWSEventBridgeTarget) pkgreconciler.Event {
+	trg.Status.InitializeConditions()
+	trg.Status.ObservedGeneration = trg.Generation
```

Comment:
This is done by Knative before the reconciliation starts already. It should be removed.
We should consider removing it from _all_ targets in the future.